### PR TITLE
Update homepage text and add logo

### PR DESCRIPTION
**Pull request by elliottower**
# Description

Text formatting changes from https://github.com/Farama-Foundation/Gymnasium/pull/471, going back to the text used in 0.28.1 which was more concise and cleaner for the homepage. 

I also made the second sentence in the description not bold so it's easier to read at a glance and see that it's a maintained fork of OpenAI Gym. We can change that back if we want though.

Before: "**The Gymnasium interface is simple, pythonic, and capable of representing general RL problems, and has a [compatibility wrapper](https://gymnasium.farama.org/content/gym_compatibility/) for old Gym environments:**"

After: "**Gymnasium is a maintained fork of OpenAI’s Gym library.** The Gymnasium interface is simple, pythonic, and capable of representing general RL problems, and has a [compatibility wrapper](content/gym_compatibility) for old Gym environments:"

Edit: I've also changed the homepage styling to use the nice graphics we use in the GitHub and do a separate header and title (one long title is very ugly and I've had multiple people tell me it should be changed)

Edit2: Changed the word 'set' to collection as that was the previously used wording, and makes the second line look better (set makes all the text fall only on the top line).

Edit3: Changed wording according to internal discussions

3: firefox_xfloGjhtNk

2: firefox_pxOD80pEOY

1: firefox_hsr5dgShr2

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Screenshots

Please attach before and after screenshots of the change if applicable.

# Checklist:

- [ ] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: gymnasium/experimental/wrappers/jax_to_torch.py [hunk 1]
```diff
@@ -1,178 +1,178 @@
...
+def _number_torch_to_jax(value: numbers.Number) -> Any:
+    """Convert a python number (int, float, complex) to a jax array."""
+    return jnp.array(value)
+
+
+@torch_to_jax.register(torch.Tensor)
+def _tensor_torch_to_jax(value: torch.Tensor) -> jnp.DeviceArray:
+    """Converts a PyTorch Tensor into a Jax DeviceArray."""
+    tensor = torch_dlpack.to_dlpack(value)  # pyright: ignore[reportPrivateImportUsage]
+    tensor = jax_dlpack.from_dlpack(tensor)  # pyright: ignore[reportPrivateImportUsage]
```
Comment (elliottower) [inline gymnasium/experimental/wrappers/jax_to_torch.py:63]: Example of the same pyright comment being done for the other usage of `from_dlpack`: `# pyright: ignore[reportPrivateImportUsage]`

File: gymnasium/experimental/wrappers/jax_to_torch.py [hunk 2]
```diff
@@ -1,178 +1,178 @@
...
+
+
+@jax_to_torch.register(jnp.DeviceArray)
+def _devicearray_jax_to_torch(
+    value: jnp.DeviceArray, device: Device | None = None
+) -> torch.Tensor:
+    """Converts a Jax DeviceArray into a PyTorch Tensor."""
+    assert jax_dlpack is not None and torch_dlpack is not None
+    dlpack = jax_dlpack.to_dlpack(value)  # pyright: ignore[reportPrivateImportUsage]
+    tensor = torch_dlpack.from_dlpack(dlpack) # pyright: ignore[reportPrivateImportUsage]
```
Comment (elliottower) [inline gymnasium/experimental/wrappers/jax_to_torch.py:94]: See above comment

File: gymnasium/experimental/wrappers/jax_to_torch.py [hunk 3]
```diff
@@ -1,178 +1,178 @@
-# This wrapper will convert torch inputs for the actions and observations to Jax arrays
```
[the remark this answers is not in the document]
Comment (elliottower) [reply gymnasium/experimental/wrappers/jax_to_torch.py:1]: Not sure why it's all red as if I changed the whole file but I just added # pyright: ignore[reportPrivateImportUsage], added some comments to this github file so you should be able to easily see the story behind it

[the remark this answers is not in the document]
Comment (elliottower) [reply gymnasium/experimental/wrappers/jax_to_torch.py:1]: Ohh I think it must've been an old version of that file, I'll pull the most recent and make sure everything matches up

[the remark this answers is not in the document]
Comment (elliottower) [reply gymnasium/experimental/wrappers/jax_to_torch.py]: Okay got it figured out, was just some weird issue with an older file

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**